bigquery-connector-common/src/main/java/com/google/cloud/bigquery/connector/common/BigQueryDirectDataWriterHelper.java

```diff
@@ -253,6 +319,13 @@ public long finalizeStream() throws IOException {
     if (this.protoRows.getSerializedRowsCount() != 0) {
       sendAppendRowsRequest();
     }
+    try {
+      checkForFailedResponse(true /* waitForResponse */);
+    } catch (BigQueryConnectorException e) {
+      throw e;
```

Comment:
We can just get rid of this catch block.

Reply:
Done.